TUTORIAL.md

```diff
@@ -738,14 +738,14 @@ of factors to consider:
 - removeProducts
 - reorderProducts
 
-Products we split into their own mutations, because the relationship is large,
+Products we split into their own mutations, because the relationship is large
 and ordered. Rules we left inline because the relationship is small, and rules
 are sufficiently minor to not have IDs.
 
-Finally, you may note that we have mutations like `addProducts` and not
-`addProduct`. This is simply a convenience for the client, since the common use
-case when manipulating this relationship will be to add, remove, or reorder more
-than one product at a time.
+Finally, you may note that we have chosen pluralized names for our mutations,
```

Comment:
Not sure about this change. "chosen pluralized names for our mutations" makes it sound like purely a naming choice/convention and not hinting that the mutations themselves offer different functionality.

Reply:
Good call. It reads as though the pluralized name is somehow creating the ability to handle more than one product at a time. The intention was to give the reader some understanding of what the sentence was calling attention to.

How about:
"Finally, you may note our product mutations act on sets of products, for example `addProducts` and not `addProduct`. This is simply a convenience for the client, since the common use case when manipulating this relationship will be to add, remove, or reorder more than one product at a time."

Reply:
Sounds good 👍